src/couch/src/couch_query_servers.erl

```diff
@@ -542,6 +542,8 @@ filter_docs(Req, Db, DDoc, FName, Docs) ->
         {ok, filter_docs_int(Db, DDoc, FName, JsonReq, JsonDocs)}
     catch
         throw:{os_process_error, {exit_status, 1}} ->
+            %% TODO: wire in csrt tracking
+            couch_stats:increment_counter([couchdb, query_server, js_filter_error]),
```

Comment:
We already have a general counter for os process errors, normal exits and error exits. Probably don't need an explicit one just for filters?

Reply:
Fixed in 4db08c9f2.